watcher/src/watcher_db.rs

```diff
@@ -445,15 +446,8 @@ impl WatcherDB {
 
     /// Get the timestamp from the watcher, or an error code,
     /// using retries if the watcher fell behind
-    /// Fog ledger server db fetcher and Ingest Server worker are both affected
-    /// by this change to move get_watcher_timestamp to a shared crate in mc
-    /// watcher
-    pub fn get_watcher_timestamp(
-        &self,
-        block_index: BlockIndex,
-        watcher_timeout: Duration,
-        logger: &Logger,
-    ) -> u64 {
+    /// The block index from ingest or ledger for example, watcher timeout set
+    pub fn get_watcher_timestamp(&self, block_index: BlockIndex, watcher_timeout: Duration) -> u64 {
```

Comment:
Sorry for not mentioning this in a previous review, but I think we should rename this to `get_block_timestamp` - it is obviously from a watcher since it sits inside `WatcherDB`.

Reply:
please see that there is already a method called get_block_timestamp. Please let me know if you want a different name

Reply:
Maybe `poll_block_timestamp` or `get_block_timestamp_polling`? What do you think @garbageslam ?

Reply:
either of those is fine with me -- slight preference for `poll_block_timestamp`?

Reply:
I have changed it to poll_block_timestamp